python/pyiceberg/table/sorting.py

```diff
@@ -112,20 +115,18 @@ class SortOrder(IcebergBaseModel):
     The order of the sort fields within the list defines the order in which the sort is applied to the data.
 
     Args:
-      order_id (int): The id of the sort-order. To keep track of historical sorting
+      order_id (int): An unique id of the sort-order of a table.
       fields (List[SortField]): The fields how the table is sorted
     """
 
-    def __init__(self, order_id: Optional[int] = None, *fields: SortField, **data: Any):
-        if order_id is not None:
-            data["order-id"] = order_id
+    order_id: int = Field(alias="order-id", default=INITIAL_SORT_ORDER_ID)
```

Comment:
Here as well, I think forcing the caller to handle sort order ID is a good idea.